sklearn/linear_model/tests/test_base.py

```diff
@@ -44,28 +45,33 @@ def test_linear_regression():
 
 def test_linear_regression_sample_weights():
     rng = np.random.RandomState(0)
+    n_samples, n_features = 6, 50  # over-determined system
 
-    for n_samples, n_features in ((6, 5), (5, 10)):
+    for fit_intercept in [True, False]:
```

Comment:
This should just be a big with `fit_intercept=True` right? But I guess it is OK to test.